interfaces/cython/cantera/base.pyx

```diff
@@ -108,8 +117,10 @@ cdef class _SolutionBase:
         cdef CxxAnyMap root
         if infile:
             root = AnyMapFromYamlFile(stringify(infile))
+            self._source = infile
         elif source:
             root = AnyMapFromYamlString(stringify(source))
+            self_source = 'custom YAML'
```

Comment:
This is supposed to be
```suggestion
            self._source = 'custom YAML'
```
right?

Reply:
absolutely! 

out of curiosity ... there appear to be some linters that work on git diffs only - in case you're using those, do you have any recommendations?

Reply:
No, no recommendations there, but it sounds like a nice feature. I use a combination of Visual Studio Code, the GitHub web interface, and `git diff` when reviewing PRs. I noticed this mainly because the syntax highlighting of `self` compared to a couple of lines above caught my attention.

Reply:
Thanks, I thought I’d ask - visual studio code is indeed nice (started using it recently, despite having been on emacs for what feels like eternities).

Anyhow, thanks for the review!